css/src/utils/flex.css

```diff
@@ -169,3 +169,183 @@
 .order-last {
   order: 6 !important;
 }
+
+/* Gap utilities */
+.gap-0 {
+  gap: 0 !important;
+}
+
+.gap-1 {
+  gap: var(--spacing-2-5) !important;
+}
+
+.gap-2 {
+  gap: var(--spacing-05) !important;
+}
+
+.gap-3 {
+  gap: var(--spacing-10) !important;
+}
+
+.gap-4 {
+  gap: var(--spacing-20) !important;
+}
+
+.gap-5 {
+  gap: var(--spacing-60) !important;
+}
+
+/* Row Gap utilities */
+.row-gap-0 {
+  row-gap: 0 !important;
+}
+
+.row-gap-1 {
+  row-gap: var(--spacing-2-5) !important;
+}
+
+.row-gap-2 {
+  row-gap: var(--spacing-05) !important;
+}
+
+.row-gap-3 {
+  row-gap: var(--spacing-10) !important;
+}
+
+.row-gap-4 {
+  row-gap: var(--spacing-20) !important;
+}
+
+.row-gap-5 {
+  row-gap: var(--spacing-60) !important;
+}
+
+/* Column Gap utilities */
+.column-gap-0 {
+  column-gap: 0 !important;
+}
+
+.column-gap-1 {
+  column-gap: var(--spacing-2-5) !important;
+}
+
+.column-gap-2 {
+  column-gap: var(--spacing-05) !important;
+}
+
+.column-gap-3 {
+  column-gap: var(--spacing-10) !important;
+}
+
+.column-gap-4 {
+  column-gap: var(--spacing-20) !important;
+}
+
+.column-gap-5 {
+  column-gap: var(--spacing-60) !important;
+}
```

Comment:
Please follow our predefined guidelines for defining the tokens and classnames. The correct classnames for `column-gap` would be using the formula (x / 4) * 100 as describes in the above comment. Please take reference from the [border](https://mds.innovaccer.com/?path=/story/styling-design-tokens-border--border) classes